Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'jekyll'
```

Comment:
Is this normal to have `jekyll` in the Gemfile of a Jekyll project? I don’t know if my question makes sense. :x